pkg/apis/scheduling/config/types.go

```diff
@@ -45,6 +47,22 @@ type QoSAwareNodeResourcesBalancedAllocationArgs struct {
 	ReclaimedResources []kubeschedulerconfig.ResourceSpec `json:"reclaimedResources,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopologyArgs holds arguments used to configure the NodeResourceTopologyMatch plugin
+type NodeResourceTopologyArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// ScoringStrategy a scoring model that determine how the plugin will score the nodes.
+	ScoringStrategy *ScoringStrategy `json:"scoringStrategy,omitempty"`
+
+	// AlignedResources are resources should be aligned for dedicated pods.
+	AlignedResources []string `json:"alignedResources,omitempty"`
+
+	// ResourcePolicy are QRMPlugin resource policy to allocate topology resource for containers.
+	ResourcePolicy consts.ResourcePluginPolicyName `json:"resourcePolicy,omitempty"`
```

Comment:
Would it be more appropriate to name this field `ResourcePluginPolicy`?

Reply:
ResourcePluginPolicy is better, done.